packages/web-components/src/components/ic-radio-group/ic-radio-group.tsx

```diff
@@ -42,6 +42,11 @@ export class RadioGroup {
    */
   @Prop() required: boolean = false;
 
+  /**
+   * The orientation of the radio buttons in the radio group. If there are more than two radio buttons in a radio group, then the orientation will always be vertical.
+   */
+  @Prop({ mutable: true }) orientation: IcOrientation = "vertical";
```

Comment:
```suggestion
  @Prop({ mutable: true, reflect: true }) orientation: IcOrientation = "vertical";
```
think this will fix orientation switching issue

Reply:
Ah thank you! I was wondering why the attribute wasn't changing